molgenis-data-row-edit/src/main/frontend/src/main.js

```diff
@@ -0,0 +1,43 @@
+import Vue from 'vue'
+import Router from 'vue-router'
+import App from './App'
+import DataRowEdit from './components/DataRowEdit'
+import i18n from '@molgenis/molgenis-i18n-js'
+import 'font-awesome/css/font-awesome.min.css'
+
+Vue.use(Router)
+
+const {lng, fallbackLng, baseUrl, dataExplorerBaseUrl} = window.__INITIAL_STATE__
+
+const dataRowEditRouter = new Router({
+  mode: 'history',
+  base: baseUrl,
+  routes: [
+    {
+      path: '/:dataTableId/:rowId',
+      component: DataRowEdit
```

Comment:
If you add props: true, you can set dataTableId and rowId as props inside your DataRowEdit component.
Saves you the hassle of doing this.$router.params... etc etc